packages/contentful--create-contentful-app/src/includeAppAction.ts

```diff
@@ -1,62 +1,61 @@
 import inquirer from 'inquirer';
-import { readFileSync, writeFileSync, mkdirSync, existsSync } from 'fs';
+import { readFileSync, writeFileSync, existsSync, renameSync } from 'fs';
 import { resolve } from 'path';
 import { CONTENTFUL_APP_MANIFEST } from './constants';
+import degit from 'degit';
+import { success, highlight } from './logger';
 
-export function cloneAppAction(templateIsTypescript: boolean, destination: string) {
-  const actionPath = resolve(
-    templateIsTypescript
-      ? 'src/app-actions/typescript/index.ts'
-      : 'src/app-actions/javascript/index.js'
-  );
+export async function cloneAppAction(templateIsTypescript: boolean, destination: string) {
+  try {
+    console.log(highlight('Cloning hosted app action.'));
+    // Clone the app actions template to the created directory under the folder 'actions'
+    const templateSource = `contentful/apps/examples/hosted-app-action-templates/${
```

Comment:
```suggestion
    const templateSource = path.join('contentful/apps/examples/hosted-app-action-templates', templateIsTypescript ? 'typescript' : 'javascript');
```

Reply:
Is this just for readability, or is there something else that it does?

Reply:
Nice, thanks!